CIP-1694/README.md

```diff
@@ -980,6 +994,26 @@ time, and instead depend on off-chain socialization to guide voters' attention t
 In this scenario, the constitutional committee might choose to only consider proposals which have
 already garnered enough votes from the DReps.
 
+### Governance action fee
+
+By introducing an optional fee on each governance action, we push the question of the DRep incentives down to the users of the system.
```

Comment:
Why cannot a deposit and fee both exist at once? why is a fee approach favoured over the deposit?